lib/OpenQA/WebAPI/Controller/API/V1/Worker.pm

```diff
@@ -206,5 +206,36 @@ sub show {
     }
 }
 
+=over 4
+
+=item delete()
+Delete worker which status is dead, and has no job.
```

Comment:
That sounds really weird, technically accurate as it may be. Elsewhere in the file the status is stated in quotes. Also, it's worth mentioning the error behavior since it's not clear what happens in other cases.

> Deletes a worker which currently has the status "dead" and no job assigned to it. An error is returned if the worker doesn't exist or has a different status.